src/pinch-zoom.js

```diff
@@ -759,7 +760,16 @@ var definePinchZoom = function () {
          */
         disable: function() {
           this.enabled = false;
+        },
+
+        /**
+         * Unmounts the zooming container and global event listeners
+         */
+        destroy: function () {
+          window.removeEventListener('resize', this.resizeHandler);
+          this.container.remove();
```

Comment:
```suggestion
          if (this.container) {
            this.container.remove();
          }
```

Reply:
That's a good suggestion. But maybe the `?.` operator will be concise in this case? 
Also, I noticed that there's no cleaning of the `this.container` property done after `.remove()`.